### DISCUSSION TITLE
What is the release schedule for connexion? 6 months since 3.1.0

### DISCUSSION THREAD

**Discussion by chrisinmtown** [spec-first/connexion#2006] [Q&A]
Please see subject. Would love to start using some of the fixes that have been merged. Thanks in advance.

**Comment by FelixSchwarz** [comment]
I think [this comment](https://github.com/spec-first/connexion/issues/1973#issuecomment-2432830773) might inform your expectations of near-term development activities.

**Comment by chrisinmtown** [comment]
Thanks @FelixSchwarz yes I saw the note about new offspring.

**Comment by chrisinmtown** [comment]
I guess I have my answer - the release schedule is purely ad hoc.
